static/js/settings_notifications.js

```diff
@@ -46,6 +53,24 @@ function rerender_ui() {
     } else {
         unmatched_streams_table.css("display", "table-row-group");
     }
+
+    // Has to be in rerender_ui because relevant rows in table are added there
+    // therefore event handler has to be set up after every call of rerender_ui
+    unmatched_streams_table.find(".unmute_stream").on("click", (e) => {
+        e.preventDefault();
+        e.stopPropagation();
+        const row = $(e.currentTarget).closest(".stream-row");
+        const stream_id = Number.parseInt(row.attr("data-stream-id"), 10);
+        const sub = sub_store.get(stream_id);
+
+        stream_settings_ui.set_muted(
+            sub,
+            !sub.is_muted,
+            row.closest(".subsection-parent").find(".alert-notification"),
+        );
+
+        update_ui_muted_stream_checkboxes(row, false);
```

Comment:
There is no need to call `update_ui_stream_checkboxes` here since it is already called in `stream_muting.update_is_muted` which is called by `stream_settings_ui.set_muted`.